### PR TITLE
Specify GatewayAccounts in API to include full list of enum for gateway's

### PR DESCRIPTION
**Pull request by XaosSintez**
- [x] Acquirers list for each gateway

### CODE REVIEW

File: spec/definitions/GatewayAccount.yaml [hunk 1]
```diff
@@ -15,9 +16,41 @@ properties:
...
+     - PayPal
+     - Payvision
+     - Plugnpay
+     - Realex
+     - RebillyProcessor
+     - Redsys
+     - Sagepay
+     - Stripe
+     - USAePay
+     - Vantiv
```
Comment (adamaltman) [inline spec/definitions/GatewayAccount.yaml]: we only have VantivLitle...

File: spec/definitions/GatewayAccount.yaml [hunk 2]
```diff
@@ -15,9 +16,41 @@ properties:
...
+     - Chase
+     - ChinaUnionPay
+     - Credorax
+     - Datacash
+     - Dengi
+     - EMerchantPay
+     - Forte
+     - GlobalCollect
+     - GlobalOne
+     - MerchantWarehouse
```
Comment (adamaltman) [inline spec/definitions/GatewayAccount.yaml]: this one has changed names to Cayan

Comment (XaosSintez) [inline spec/definitions/GatewayAccount.yaml]: But i still see old name in backend...

File: spec/definitions/GatewayAccount.yaml [hunk 3]
```diff
@@ -15,9 +16,46 @@ properties:
...
+     - EMerchantPay
+     - Forte
+     - GlobalCollect
+     - GlobalOne
+     - iCheque
+     - Ilixium
+     - MerchantWarehouse
+     - Moneris
+     - NMI
+     - Ochapay
```
Comment (slavcodev) [inline spec/definitions/GatewayAccount.yaml]: The brand name is `OchaPay`, I think others also need to verify, AFAIK we decided to use consistent a real brand names.

**Comment by adamaltman** [comment]
Looks good

**Pull request merged**